Adding release notes for ocis 4.0.3 and 4.0.4 (#4914)
owncloud/docs: modules/ROOT/pages/ocis_release_notes.adoc

toc::[]

== Infinite Scale 4.0.4

[discrete]
=== General

This is a patch release only, please update as soon as possible.

[discrete]
=== Enhancement

> * Update reva to improve trashbin listing: https://github.com/owncloud/ocis/pull/7858[#7858]

Reviewer: ```suggestion
* Update Reva to improve trash bin listing: https://github.com/owncloud/ocis/pull/7858[#7858]
```

== Infinite Scale 4.0.3

[discrete]
=== General

This is a patch release only, please update as soon as possible.

[discrete]
=== Issues Fixed

> * Bump reva to 2.16.1. https://github.com/owncloud/ocis/pull/7350[#7350]

Reviewer: ```suggestion
* Bump Reva to 2.16.1. https://github.com/owncloud/ocis/pull/7350[#7350]
```

* We fixed a problem where the states of received shares were reset to PENDING in the +
`ocis migrate rebuild-jsoncs3-indexes` command. https://github.com/owncloud/ocis/issues/7319[#7319]

> * We fixed an issue that allowed to create two schools with the same school number.  https://github.com/owncloud/ocis/pull/7351[#7351]

Reviewer 2: ```suggestion
* We fixed an issue that allowed two schools to be created with the same school number.  https://github.com/owncloud/ocis/pull/7351[#7351]
```

> * Disable username validation for keycloak example. https://github.com/owncloud/ocis/pull/7230[#7230] +

Reviewer: ```suggestion
* Disable username validation for Keycloak example. https://github.com/owncloud/ocis/pull/7230[#7230] +
```

> Set `GRAPH_USERNAME_MATCH` to `none`. To accept any username that is also valid for keycloak.

Reviewer 2: ```suggestion
Set `GRAPH_USERNAME_MATCH` to `none` to accept any username that is also valid for keycloak.
```

> Set `GRAPH_USERNAME_MATCH` to `none` to accept any username that is also valid for keycloak.

Reviewer: ```suggestion
Set `GRAPH_USERNAME_MATCH` to `none` to accept any username that is also valid for Keycloak.
```

> * Actually pass `PROXY_OIDC_SKIP_USER_INFO` option to oidc client middleware.  https://github.com/owncloud/ocis/pull/7220[#7220]

Reviewer: ```suggestion
* Actually pass `PROXY_OIDC_SKIP_USER_INFO` option to OIDC client middleware.  https://github.com/owncloud/ocis/pull/7220[#7220]
```
